src/pip/_internal/metadata/importlib/_dists.py

```diff
@@ -33,6 +34,11 @@
 from ._compat import BasePath, get_dist_name
 
 
+@functools.lru_cache(maxsize=1024)
+def requirements(req_str: str) -> Requirement:
```

Comment:
Should this be `requirement` instead? Or `create_requirement` or `requirement_from_string`.

Reply:
I changed to `create_requirement`, but I don't have strong feelings on it.

Reply:
As long as it’s not a plural…